### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by brahami14** [yanx27/2DPASS#50]
Hello, 
Thank you for your great work , 

the models you put in google drive, they were trained with an additional validation set and  using instance-level segmentation ?

Thanks in advance

**Comment by yanx27** [comment]
Hi @brahami14 , all models in Google Drive are trained on the **pure training set** with corresponding config files (only large model on SemanticKITTI use instance augmentation). To gain the higher performance, we use trained models to generate pseudo labels on the validation set, and fine-tune on these labels.
